internal/http/server.go

```diff
@@ -172,6 +172,7 @@ func NewServer(addrs []string, handler http.Handler, getCert certs.GetCertificat
 			MinVersion:               tls.VersionTLS12,
 			NextProtos:               []string{"http/1.1", "h2"},
 			GetCertificate:           getCert,
+			ClientAuth:               tls.RequestClientCert,
```

Comment:
does this not break anything? @aead for regular TLS clients ?

Reply:
Should not. See: https://pkg.go.dev/crypto/tls#ClientAuthType

Also, without requesting a client certificate it will not work.